pkgs/tools/misc/tmux/default.nix

```diff
@@ -23,7 +23,7 @@ stdenv.mkDerivation rec {
   pname = "tmux";
   version = "3.1";
 
-  outputs = [ "out" "man" ];
+  outputs = [ "out" ];
```

Comment:
I think this line is redundant if you only want to have the `"out"` output.

Reply:
I dom't believe removing the man output is the solution.
I am not sure how nix-shell and MANPATH interact
 `nix-shell --pure -p tmux.man man` does not find the man but my installed tmux brings up the manpage

Reply:
> I think this line is redundant if you only want to have the `"out"` output.

Oh yes, I agree. I'll change this in a second.

Reply:
> I dom't believe removing the man output is the solution.
> I am not sure how nix-shell and MANPATH interact
> `nix-shell --pure -p tmux.man man` does not find the man but my installed tmux brings up the manpage

Okay, weird, it doesn't work for me. But it's not terribly important. I'll await further guidance how to proceed. :-) Thank you two of your review!